load-jetpack.php

```diff
@@ -68,7 +68,7 @@ function jetpack_should_use_minified_assets() {
 	require_once JETPACK__PLUGIN_DIR . 'class.jetpack-admin.php';
 	require_once JETPACK__PLUGIN_DIR . 'class.jetpack-affiliate.php';
 	$jitm = new Automattic\Jetpack\JITM();
-	$jitm->register();
+	add_action( 'plugins_loaded', array( $jitm, 'register' ) );
```

Comment:
I am not sure this should live here as well. I would probably move it somewhere where it makes more sense but am not sure where.

Reply:
This seems to break JITMs 

When looking in the Network console, I see results from the JITM REST endpoint that suggests there should be a JITM showing, but it does not. 

![image](https://user-images.githubusercontent.com/7129409/68865520-f315c880-06c0-11ea-9ba9-6f49265d6ea8.png)

Reply:
I tested this by changing jetpack 7.9 and it worked for me as expected. 
Can you retest it?